public/popup.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Getting Started Extension's Popup</title>
```

Comment:
Maybe we can change the title to something like GrowthBook DevTools? Is this popup functional?